<!DOCTYPE html>
<html>
  <head>
    <title>Geolocation Test: call geolocation.clearWatch with number arguments, no exception expected</title>
    <link rel="author" title="Intel" href="http://www.intel.com/">
    <link rel="help" href="http://www.w3.org/TR/2012/PR-geolocation-API-20120510/#geolocation">
    <link rel="help" href="http://www.w3.org/TR/2012/PR-geolocation-API-20120510/#watch-position">
    <script src="/resources/testharness.js"></script>
    <script src="/resources/testharnessreport.js"></script>
  </head>
  <body>
    <div id="log"></div>
    <script>
      var t = async_test("Test checks that call geolocation.clearWatch with number arguments, no exception expected");
      setup({timeout: 20000});
      try {
        t.step(function () {
          navigator.geolocation.clearWatch(1236);
        });
        t.done();
      } catch (err) {
        t.step(function () {
          assert_unreached("Error message: " + err.message);
        });
        t.done();
      }
    </script>
  </body>
</html>
